js/src/wrappers/claude-agent-sdk.test.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  test,
+  expect,
+  describe,
+  beforeEach,
+  beforeAll,
+  afterEach,
+} from "vitest";
+import { wrapClaudeAgentSDK, wrapClaudeAgentQuery } from "./claude-agent-sdk";
+import { initLogger, _exportsForTestingOnly } from "../logger";
+import { configureNode } from "../node";
+import { z } from "zod";
+
+// Try to import the Claude Agent SDK - skip tests if not available
+let claudeSDK: unknown;
+try {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore - SDK may not be installed
+  claudeSDK = await import("@anthropic-ai/claude-agent-sdk");
```

Comment:
For testing environment we should probably just assume the dependency is installed. I know we've used makefile to conditionally install them. If we want to continue that practice, we should just exclude this file from the sdk core test run and manually install (finally remove) and then run this test. 

Personally, I would rather just add the dev dependency and assume the dep exists and minimize the complexity.